db/schema.rb

```diff
@@ -134,12 +139,13 @@
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
+    t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true, using: :btree
+    t.index ["email"], name: "index_users_on_email", unique: true, using: :btree
+    t.index ["legal_agreement"], name: "index_users_on_legal_agreement", using: :btree
+    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
+    t.index ["username"], name: "index_users_on_username", unique: true, using: :btree
```

Comment:
I wonder if changing the indexes to this will break the migrations. Most of the indexes weren't created with the migrations.